Simulation/DetSimSD/src/TrackerCombineSensitiveDetector.cpp

```diff
@@ -23,6 +23,8 @@ void TrackerCombineSensitiveDetector::Initialize(G4HCofThisEvent* HCE){
 
 G4bool TrackerCombineSensitiveDetector::ProcessHits(G4Step* step, G4TouchableHistory*){
   dd4hep::sim::Geant4StepHandler h(step);
+  if (fabs(h.trackDef()->GetPDGCharge()) < 0.01) return true;
```

Comment:
Hi @fucd,
Is this expected?

Reply:
@mirguest Yes, this is one step same as CDR. More reasonable condition will be test/considered continuely.